app/models/target.rb

```diff
@@ -0,0 +1,8 @@
+class Target < ApplicationRecord
+  belongs_to :topic
+  validates :tittle, presence: true
```

Comment:
```suggestion
  validates :title, presence: true
```

Reply:
You right! a little mistake hehe. i'll fix it!